src/scene/materials/standard-material-options.js

```diff
@@ -20,6 +20,9 @@ import { LitOptions } from "./lit-options.js";
  * value.
  * @property {boolean} opacityTint Defines if {@link StandardMaterial#opacity} constant should affect opacity value.
  * @property {boolean} packedNormal If normal map contains X in RGB, Y in Alpha, and Z must be reconstructed.
+ * @property {boolean} glossInvert Invert the gloss channel.
```

Comment:
These comments could explain relation to roughness.

Reply:
explanation was added to standard-material jsdoc